lib/main.js

```diff
@@ -1,4 +1,4 @@
-var Hls = require("hls.js");
-var attachVideojsStreamrootProvider = require('./videojs5-hlsjs-source-handler.js');
+var Hlsjs = require('hls.js');
+var attachVideojsHtml5Hlsjs = require('./videojs-hlsjs-source-handler.js');
 
-attachVideojsStreamrootProvider(window, window.videojs, Hls);
+attachVideojsHtml5Hlsjs(window.videojs, Hlsjs);
```

Comment:
Since we are NPM'ing maybe you can export this function in case the `videojs` is in a NPM bundle and needs to be manually injected.